### PR TITLE
feat(dop): DOP adds batch update CMS API

### PR DESCRIPTION
**Pull request by kakj-go**
#### What this PR does / why we need it:
The project level pipeline adds global parameter configuration and lacks update interface

#### Which issue(s) this PR fixes:

- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/gantt?filter__urlQuery=eyJpdGVyYXRpb25JRHMiOlsxMTY0XSwiYXNzaWduZWUiOlsiMTAwMDU2MCJdfQ%3D%3D&id=299271&iterationID=1164&pId=293275&type=TASK)

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |       DOP adds batch update CMS API       |
| 🇨🇳 中文    |      DOP添加批量更新CMS API        |

### CODE REVIEW

File: modules/openapi/api/apis/dop/adaptor_cicd_config_del.go [hunk 1]
```diff
@@ -1,57 +0,0 @@
...
-
-var ADAPTOR_CICD_CONFIG_DEL = apis.ApiSpec{
-	Path:        "/api/cicds/configs",
-	BackendPath: "/api/cicds/configs",
-	Host:        "dop.marathon.l4lb.thisdcos.directory:9527",
-	Scheme:      "http",
-	Method:      "DELETE",
-	CheckLogin:  true,
-	Doc:         "summary: 删除Pipeline指定命名空间下的某个配置",
-	Audit: func(ctx *spec.AuditContext) error {
```
Comment (sfwn) [inline modules/openapi/api/apis/dop/adaptor_cicd_config_del.go:33]: how to do audit in your new code?

**Review by sfwn — CHANGES_REQUESTED** [review]

**Comment by kakj-go** [comment]
audit image:

image

File: api/proto/dop/cms/cicdcms.proto [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+}
+
+message Config {
+  string comment = 1;
+  bool encrypt = 2;
+  string key = 3;
+  string type = 4;
+  string value = 5;
+}
+
```
Comment (sfwn) [inline api/proto/dop/cms/cicdcms.proto:89]: remove useless blank lines.

File: modules/dop/providers/cms/cicdcmsservice.go [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+		NamespaceName: req.NamespaceName,
+		AppID:         req.AppID,
+		Encrypt:       req.Encrypt,
+		Configs:       req.Configs,
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	return &pb.CICDCmsCreateResponse{
```
Comment (sfwn) [inline modules/dop/providers/cms/cicdcmsservice.go:142]: do you need add audit for create?

**Review by sfwn — CHANGES_REQUESTED** [review]

**Comment by sfwn** [comment]
> audit image:
> 
> image

删除了流水线配置
设置了流水线配置

**Review by sfwn — APPROVED** [review]

**Pull request merged**